### PR TITLE
Update version: Streetwriters.Notesnook version 2.6.14

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135209)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-22T22:15:33.4724200Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 628.)

File: manifests/s/Streetwriters/Notesnook/2.6.14/Streetwriters.Notesnook.installer.yaml [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.14/notesnook_win_x64.exe
+  InstallerSha256: A79D93598D4C5E624D0B7751DD57705BA9B3C270BED90BB3174817CC0DF0DE97
+- Architecture: x64
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.14/notesnook_win_x64.exe
+  InstallerSha256: A79D93598D4C5E624D0B7751DD57705BA9B3C270BED90BB3174817CC0DF0DE97
+- Architecture: arm64
+  InstallerUrl: https://github.com/streetwriters/notesnook/releases/download/v2.6.14/notesnook_win_arm64.exe
+  InstallerSha256: AA2554DEE2666AC711D00469CF3A993F397C23E8BE831F987976506494B987FB
```
Comment (stephengillie) [inline manifests/s/Streetwriters/Notesnook/2.6.14/Streetwriters.Notesnook.installer.yaml:21]:
```suggestion
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**